src/wasm-interpreter.h

```diff
@@ -578,6 +578,7 @@ class ModuleInstanceBase {
         }
         case f32: return Literal(load32u(addr)).castToF32();
         case f64: return Literal(load64u(addr)).castToF64();
+        case v128: assert(false && "v128 not implemented yet");
```

Comment:
We should implement these ASAP, as the interpreter should be complete. But ok to leave to a later PR if you prefer.